File: hotelbediax-frontend/src/types/destination.ts
export type DestinationType = 'Ciudad' | 'Playa' | 'Montaña' | 'Cultural' | 'Aventura';

export interface Destination {
  id: number;
  name: string;
  description: string;
  countryCode: string;
  type: DestinationType;
  lastModification: string;
}

File: hotelbediax-backend/HotelBediaX.Api/Models/Destination.cs
namespace HotelBediaX.Api.Models;

public class Destination
{
    public int Id { get; set; }
    public string Name { get; set; } = "";
    public string Description { get; set; } = "";
    public string CountryCode { get; set; } = "";
    public DestinationType Type { get; set; }
    public DateTime LastModification { get; set; }
}
